avr/platform.txt

```diff
@@ -127,6 +147,9 @@ tools.dfu-programmer.upload.params.quiet=
 tools.dfu-programmer.upload.noverify=
 tools.dfu-programmer.upload.pattern=/bin/sh -c '"{cmd.path}" atmega32u4 erase && "{cmd.path}" atmega32u4 flash "{build.path}/{build.project_name}.hex" && "{cmd.path}" atmega32u4 start'
 
+tools.rm.cmd=rm -f
+tools.java.cmd.windows=del /F
```

Comment:
Should this be `tools.rm.cmd.windows` (instead of `tools.java.cmd.windows`)?

Reply:
Thanks. I just force-pushed a fix.